src/country.js

```diff
@@ -0,0 +1,72 @@
+import axios from "axios";
+
+async function fetchCountry(name) {
+  const URI = 'https://restcountries.com/v2/';
+  const ENDPOINT = 'name/'
```

Comment:
Variabelen in camelCase schrijven
```suggestion
  const uri = 'https://restcountries.com/v2/';
  const endpoint = 'name/';
```